apps/lobe-chat/1.36.19/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.36.19
+    image: lobehub/lobe-chat:v1.36.25
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only difference between the two lines is the version of the Lobe-Chat image being used:

```yaml
-    image: lobehub/lobe-chat:v1.36.19
+    image: lobehub/lobe-chat:v1.36.25
```

This change should not introduce any functional issues but simply updates the running version of the service to v1.36.25, which might contain new features or bug fixes. There are no major irregularities or optimizations necessary at this time.